code/modules/mob/living/living.dm

```diff
@@ -661,6 +661,8 @@
 				var/mob/living/carbon/human/H = src
 				if(H.dna.species.blood_color)
 					existing_trail.color = H.dna.species.blood_color
+			else if(isalien(src))
+				existing_trail.color = "#00FF32"
```

Comment:
You should try with this color, this is the color used by other xeno blood stuff
```suggestion
				existing_trail.color = "#05EE05"
```